src/databricks/labs/ucx/workspace_access/base.py

```diff
@@ -57,3 +59,14 @@ def noop():
                 pass
 
             return partial(noop)
+
+    def response_to_request(self, acls: Optional["list[AccessControlResponse]"] = None) -> list[AccessControlRequest]:
```

Comment:
it does not belong to this class. it only belongs to GenericPermissionsSupport. this has to be a private method.